docs/DeveloperGuide.md

```diff
@@ -171,6 +171,46 @@ As such, the detailed descriptions for the Address Book subsystem above can be t
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Tag Management
+
+This section will describe tag management in the address book as well as the features implemented.
+
+### Centralising Tags in the Address Book
+
+In the previous implementation, all `Tag`s are independent of one another despite having the same tag names. Hence, centralising the tags enable the user to easily manipulate them as well as easily searching for `Person`s that contain the tag.
```

Comment:
I think when adding the `Tag` or `Person`, try not to follow it up with a "s" behind cos when it is rendered it looks a bit weird.
<img width="65" alt="Screenshot 2022-03-21 at 10 42 16 PM" src="https://user-images.githubusercontent.com/13099969/159285397-a8171d95-61c0-4d6f-b2bc-fc86f7bf32dc.png">
 
I can suggest referring it to as `Tag` objects or `Person` objects

Reply:
I realise this applies to many other places in this PR so I'm not gonna individually highlight each of them lol if you decide to alter this

Reply:
Thank you for your feedback on this! I have followed your suggestion and amended the changes in 61207721